### ISSUE TITLE
Enrich Getting Started with first steps

### ISSUE THREAD

**Issue by asys3** [viadee/vPW#36]
Just got vpw up and running with docker. Works like a charm out of the box! Awesome.
Many thanks to make vpw Opensource!

I just started vpw via http://localhost:8080 but don't see any process model. 
Where to select it?
Perhaps anything else to configure in Kafka, pulling client etc. ?

I would expect a getting started to not only include a system startup (what really seem to work!) but also the steps from starting instances to visualize them in vpw.
Perhaps I missed sth. here and it is easy to add for you. Would be nice to have it!

[Videos on product website](https://www.viadee.de/en/solutions/business-process-management/process-warehouse) start when a process is model is already configured and can be displayed in vpw.

**Comment by mschulte** [comment]
Hey @asys3,

thanks for your feedback! 

We could definately add a "getting started" from an end-user-perspective (not only getting all things wired up together).
Hope to find time for that soon. As always: PRs are much appreciated ;)

For the meantime a short guide (which i will reuse for "official" docs of course):
1. Fire up your browser on http://localhost:8080 --> you should see an empty process warehouse-dashboard
2. Click "plus" and select your process definition you want to add a visualisation for:
![grafik](https://user-images.githubusercontent.com/843635/153497042-e10df9b7-f7b2-4930-9854-5b13e31579c0.png)
3. You should see the process modell of your selected process definition
4. Click "plus" on that process definition to select so called indicators on process/activity level 
![grafik](https://user-images.githubusercontent.com/843635/153497423-253789cc-78e0-4fff-b611-9f3d3e6759c2.png)

Hope that helps in the first place?

**Comment by mschulte** [comment]
Of course it is. It seems your vpw-pipeline is not running. Therefore all events that are produced by camunda still reside in kafka topics but are not consumed by the vpw/ transported to elasticsearch.

Could you provide a excerpt from your docker/container logs that may give us a hint why your pipeline crashed?
What are your steps to reproduce the problem?
Normally a "git checkout" follwed by "docker-compose up -d" should be enough to have a working demo-environment.

**Comment by DaZang** [comment]
Thank you for your valuable feedback @asys3 

The vpw-pipeline only (re)tries to reach the analyzer for a bit more than one minute, which can be an issue when using vpw on a slower notebook with docker-compose. We are going to provide a fix for that really soon.

If that really is the issue, something you could try in the meantime is:

1. Shut everything down again `docker-compose down`
2. Start analyzer-postgres first: `docker-compose up -d analyzer-postgres`
3. Wait a couple of seconds until the container ist up
4. Start vpw-analyzer next: `docker-compose up -d vpw-analyzer`
5. Wait a minute (or until `docker ps -a` shows the "vpw-analyzer" containers status as "Up..."
6. Start everything else with `docker-compose up -d`
7. Wait until everything is up and running and try again to select the process definition.

Also providing the logs as @mschulte suggested, would be a great help.

**Comment by asys3** [comment]
It seems a timing problem for me, actually. I always started from scratch (container & volumes pruned) and come to the conclusion that heavy memory consumption/ cpu load seem to be a possible error case (my laptop has 64GB  i5-1135G7, but with additional virtualbox running).
So with concentrating only starting vpw containers i got it running.
But still can reproduce the problem. So see the log output from `docker-compose -up` attached that includes my error reported above
[all.logs.tar.gz](https://github.com/viadee/vPW/files/8050106/all.logs.tar.gz)


. Additionally sometimes I saw a Bad Gateway error appearing:



![vpwBadGateway](https://user-images.githubusercontent.com/7109324/153629663-bbaa7e91-7675-4e4a-944a-c76b6684f081.png)

**Comment by mschulte** [comment]
Hey Uwe,

we tackled this be longer retry-time-cycle for importing process models. 
Docs are also extended. Cf. https://github.com/viadee/vPW/pull/68

We will release this with the next version probably on friday this week.

Could you then verify if the issue still persists? I would like to close this issue after merging the above mentioned pullrequest.

Again: Thanks for your feedback!

Kind regards,
Matthias

**Comment by asys3** [comment]
The documentation is a lot better now - great work!
Should help everyone to start it up quickly.

The error situation seems not solved for me though.
Again I started my virtualbox machine with Win10 in parallel (16Gb Mem assured) on my laptop (32Gb Mem) and that must be insufficient memory for vpw to work successfully. It even crashes my virtual machine :-( 
The result is that a model can't be selected (empty selectbox / Bad gateway - see error above).
Here are the logs.
[allStartupLogs.txt.tar.gz](https://github.com/viadee/vPW/files/8456284/allStartupLogs.txt.tar.gz)
I then stopped the container and my virtualbox instance and restarted the stopped containers.
Still the same error.
Logs here:
[allStartupLogsNoVM.txt.tar.gz](https://github.com/viadee/vPW/files/8456290/allStartupLogsNoVM.txt.tar.gz)
After that I pruned all containers, volumes and tried from scratch again with no virtualbox instance running.
That seem to work reliable.
The logs to compare are here.
[allStartupLogsNoVMPrune.txt.tar.gz](https://github.com/viadee/vPW/files/8456299/allStartupLogsNoVMPrune.txt.tar.gz)
There are still some errors inside (for example: ZooKeeper not started) but that seems only temporary.

So from my side I would say that's a memory problem. 16GB main mem is to low for vpw.
If you start from scratch with 32GB everything works.

**Comment by DaZang** [comment]
Hey Uwe,
as you wrote your comment one day after our newest release: 

Can you try to remove the hashtag in the line below to restrict java heapspace for elastic https://github.com/viadee/vPW/blob/5736341d4dcc829360b66ea25033b288933a5605/docker-compose.yml#L52

**Comment by asys3** [comment]
Hi David,
that helps a lot. So even if I have my virtual machine running parallel to vPW it works as desired.
So limiting Elastic would help in my situation. Again a hint that low memory at runtime is the reason.
Don't know if this activated limitation would be a good starting point or is it generally a limit hat is to hard for other users/usecases?
Anyway. Thanks for the hint with the switch!

**Comment by DaZang** [comment]
Hi Uwe,
nice knowing that it solved the problem for you!

I guess the answer lies somewhere in the middle. But for just trying it out on a local machine the 1GB Limit would be a fair starting point. I think we will use this as a default in the future!
